build/deps/gen/dep_rules_python.bzl

```diff
@@ -3,9 +3,9 @@
 load("@//:build/http.bzl", "http_archive")
 
 TAG_NAME = "0.40.0"
-URL = "https://api.github.com/repos/bazelbuild/rules_python/tarball/0.40.0"
-STRIP_PREFIX = "bazelbuild-rules_python-1944874"
-SHA256 = "3e8396b0741231f07e175d0e721969ea3456b4330a4daf32d1520ce427a09937"
+URL = "https://github.com/bazel-contrib/rules_python/releases/download/0.40.0/rules_python-0.40.0.tar.gz"
```

Comment:
This now uses a user uploaded file rather than the tarball provided by Github. On theory, this is unsafe.

Reply:
I've verified that the file content is the same, using both `diff -r` and `sha256sum`

Reply:
It's also not the mtime btw, Github sensibly sets that to `0`.